src/ClusterBootstrap/template/kubeconfig.yaml.template

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Is this a copy of "src/ClusterBootstrap/template/kubelet/worker-kubeconfig.yaml.template" ?

Reply:
Not really, we recently changed cluster scaler to local usage only so the cert path will be different

Reply:
can you create a sub-folder under src/ClusterBootstrap/template and put this file to the sub-folder?

Reply:
yes!

Reply:
Fixed! Thanks!